src/translations/en.json

```diff
@@ -123,5 +123,96 @@
+    "queenDescription": "Protects your other cards from being targeted by the effects of other cards. This protects your cards against 2's (both effects), 9's, and Jacks, but not scuttling.",
+    "jack": "Jack",
+    "jackDescription": "Play on an opponent's point card to steal it. Point card returns to opponent if the Jack is scrapped or if another Jack is used to steal it back.",
+    "oneoffsTitle": "One-Offs",
+    "oneoffsText": "Number cards (except 8's and 10's) can be played for a One-Off effect, which scraps the card for an effect based on the rank of the card played. Whenever a one-off is played, the other player may counter it using a two to cancel the effect.",
+    "ace": "Ace",
+    "aceDescription": "Scrap all point cards on the field.",
+    "twoE1": "Two - Effect 1",
+    "twoE1Description": "Two's have two alternative one-off effects: Counter target One-Off Effect. May be played during your opponent's turn.",
+    "twoE2": "Two - Effect 2",
+    "twoE2Description": "Twos have two alternative one-off effects: Scrap target Royal or Glasses Eight.",
+    "three": "Three",
+    "threeDescription": "Choose a card from the scrap pile and put it in your hand.",
+    "four": "Four",
+    "fourDescription": "Your opponent discards two cards of their choice.",
+    "five": "Five",
+    "fiveDescription": "Draw two cards from the deck up to the 8 card hand limit.",
+    "six": "Six",
+    "sixDescription": "Scrap all Royals and Glasses Eights on the field.",
+    "seven": "Seven",
+    "sevenDescription": "Reveal the top two cards from the deck and choose one to play however you choose.",
+    "nine": "Nine",
+    "nineDescription": "Return a card from your opponent's field to their hand. They cannot play it next turn.",
+    "multiplayerTitle": "Multiplayer Variants",
+    "multiplayerText1": "Presently only 2-player cuttle is available at ",
+    "multiplayerText2": "however Cuttle can be played with 2-4 players. 3 and 4-player Cuttle each have slight variations on the rules to optimize the game balance while changing the pace and the feel. Give them a try next time you're playing Cuttle in a group!",
+    "3PTitle": "3 Player Cuttle: Cutthroat",
+    "3PText": "Cutthroat is a brutally fast-paced 1v1v1 version of Cuttle, full of betrayal and pivotal moments!",
+    "3PRules": "The rules are the same as above with the following changes:",
+    "3PRule1": "The win condition is reduced. Players need 14 points to win by default, 9 points with one King in play, 5 points with two Kings in play, and having 3 Kings in play immediately wins (goal becomes zero).",
+    "3PRule2": "To start play, deal each player 5 cards and the player to the left of the dealer goes first. Which player is the dealer rotates clockwise each game.",
+    "3PRule3": "Shuffle two Jokers into the deck. Jokers count as Royals with the effect \"Steal target Royal\".",
+    "3PRule4": "The hand limit is reduced to 7 (from 8).",
+    "4PTitle": "4 Player Cuttle: 2v2",
```

Comment:
@Rafajrg21 what are your thoughts on whether to use a prefix in sibling keys like you have here for 3P and 4P sections, vs adding an additional layer to the dictionary like:
```
"rules": {
  "3P": {
    "title": "3 Player Cuttle: Cutthroat"
     // etc
  }
}
```

Which do you prefer and why?

Reply:
Hi, so I first thought of only adding a prefix so every page had it translation layer but right after I pushed the change I said to myself "I should've added another level for each section" so it would result in something like this:
```json
"rules": {
  "3P": {
    "title": "3 Player Cuttle: Cutthroat"
     "...etc"
  },
  "4P": {
    "title": "4 Player Cuttle: 2v2"
     "...etc"
  },
  "faq": {
    "title": "FAQ"
     "...etc"
  },
}
```
Adding a "royals" and "one-offs" section as well. I think this approach enhances the Developer Experience so I can update the PR with this change and the string literal correction. What do you think about the royals and one-offs sections?

Reply:
Yes I think the additional layering makes sense and will work nicely for the other sections as you've suggested. Good call

Reply:
Resolved in commit 84cb820